src/maven-repair/src/main/java/com/github/tdurieux/repair/maven/plugin/NPEFixMojo.java

```diff
@@ -192,7 +196,7 @@ public void execute() throws MojoExecutionException {
             jsonObject.write(writer);
             writer.close();
         } catch (Exception e) {
-            throw new RuntimeException(e);
+            Log.info("We don't break the build for errors during the build, continue.");
```

Comment:
log the exception, be more explicit, elaborate in 2-3 sentences

Reply:
Tied to explain a bit more, not sure should I print e.printStackTrace();